scripts/template_matcher.py

```diff
@@ -30,24 +42,21 @@ def predict(self, img):
         """
         visual_diff = {}
 
-        # TODO: get keypoints and descriptors from input image using SIFT
-        #       store keypoints in variable kp and descriptors in des
+        # compute keypoints and descriptors for the scene
+        kp, des = self.sift.detectAndCompute(img,None)
 
         for k in self.signs.keys():
             #cycle trough templage images (k) and get the image differences
             visual_diff[k] = self._compute_prediction(k, img, kp, des)
-
-        if visual_diff:
-            pass
-            # TODO: convert difference between images (from visual_diff)
-            #       to confidence values (stored in template_confidence)
+        
+        if not (None in visual_diff.values()):
+            template_confidence = {}
+            for k in visual_diff:
+            	template_confidence[k] = 50/visual_diff[k]
```

Comment:
Where did 50 come from?  It would have been cool to have some comments to explain this (even something like, this was arbitrary and it worked)